enterprise/api_client/lms.py

```diff
@@ -236,6 +236,26 @@ def enroll_user_in_course(self, username, course_id, mode):
             }
         )
 
+    def unenroll_user_from_course(self, username, course_id):
+        """
+        Call the enrollment API to unenroll the user in the course specified by course_id.
+        Args:
+            username (str): The username by which the user goes on the OpenEdx platform
+            course_id (str): The string value of the course's unique identifier
+        Returns:
+            bool: Whether the unenrollment succeeded
+        """
+        enrollment = self.get_course_enrollment(username, course_id)
```

Comment:
is there a reason we need to call the api to check if the enrollment exists before unenrolling? how does the enrollment api handle a call to unenroll someone from a course they're not enrolled in? i think it would be better to make one call instead of two.